app/helpers/ems_middleware_helper/textual_summary.rb

```diff
@@ -56,4 +56,14 @@ def textual_topology
      :link  => url_for_only_path(:controller => 'middleware_topology', :action => 'show', :id => @record.id),
      :title => _('Show topology')}
   end
+
+  def textual_authentication_status
+    auth_status = @record.authentication_for_summary.first
+
+    if auth_status[:status_details].blank?
+      auth_status[:status]
+    else
+      "#{auth_status[:status]} - #{auth_status[:status_details]}"
```

Comment:
How is i18n handled for this?

Reply:
The `status_details` text comes from the provider. In the case of Hawkular, there are some strings [here](https://github.com/ManageIQ/manageiq-providers-hawkular/blob/16ea270db1be953a929a078241ee14cacdc06673/app/models/manageiq/providers/hawkular/middleware_manager.rb#L33-L52). They aren't translated. I think I could make them translatable by submitting a PR to apply `_( ...whatever... )` to the strings. The only case that cannot be translated is when the [message of the exception is passed through](https://github.com/ManageIQ/manageiq-providers-hawkular/blob/16ea270db1be953a929a078241ee14cacdc06673/app/models/manageiq/providers/hawkular/middleware_manager.rb#L45).

On the other hand, the `status` field values are defined in the [AR authentication model](https://github.com/ManageIQ/manageiq/blob/master/app/models/authentication.rb#L41-L49). These strings are unstranslated and seems like they cannot be translated because there are some checks using the strings.